app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(human["name"], human["age"]) for human in people]
+    for human in people:
+        if human.get("wife"):
+            husband = Person.people[human["name"]]
+            wife = Person.people[human["wife"]]
+            husband.wife = wife
+            wife.husband = husband
```

Comment:
nothing said that if "wife" we need to set husband. Add additional check for husband. We need to set values due to input data

Reply:
It is logical how you wrote it, but more readable code will be how Arsen suggested it. Rewrite please